packages/dictionary/src/metaSchema/dictionarySchemas.ts

```diff
@@ -66,14 +66,23 @@ export type SchemaFieldValueType = zod.infer<typeof SchemaFieldValueType>;
 /* ****************************** *
  * Field Type Restriction Objects *
  * ****************************** */
-export const StringFieldRestrictions = zod
+export const BooleanFieldRestrictions = zod.object({ required: zod.boolean() }).partial();
```

Comment:
This section looks heavily modified but its just organizing the Restriction schemas alphabetically.